tests/tests/Converter/ObjectTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace MarcinOrlowski\ResponseBuilder\Tests;
+
+/**
+ * Laravel API Response Builder
+ *
+ * @package   MarcinOrlowski\ResponseBuilder
+ *
+ * @author    Marcin Orlowski <mail (#) marcinOrlowski (.) com>
+ * @copyright 2016-2020 Marcin Orlowski
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      https://github.com/MarcinOrlowski/laravel-api-response-builder
+ */
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Config;
+use MarcinOrlowski\ResponseBuilder\Converter;
+use MarcinOrlowski\ResponseBuilder\Converters\ToArrayConverter;
+use MarcinOrlowski\ResponseBuilder\Exceptions as Ex;
+use MarcinOrlowski\ResponseBuilder\ResponseBuilder as RB;
+use MarcinOrlowski\ResponseBuilder\Tests\Converters\FakeConverter;
+use MarcinOrlowski\ResponseBuilder\Tests\Models\TestModel;
+use MarcinOrlowski\ResponseBuilder\Type;
+
+class ObjectTest extends TestCase
+{
+	/**
+	 * Checks if convert returns data without any wrapper 'key' when it's set to be empty
+	 */
+	public function testConvertWithEmptyKeyInConfig(): void
+	{
+        // custom converter with it's own response structure
+        $fake = new FakeConverter();
+
+        $data = collect([1, 2, 3]);
+
+        $possibleEmptyKeyValues = ['', 0, '0', false, NULL];
```

Comment:
Looks good, but I now wonder if this is a good idea to allow that much ways of setting empty key. What if we will be strict here and to make it empty you need to set it to `null` only? That'd require changes in main code as we need to ensure that key is either a string or null (anything else should throw an exception I believe + additional test to ensure it works that way). Also change in main code to check against `null` instead of using `isset()` would be required. But that would make the code less error prone

Reply:
Thanks, I agree on having just `null` as the empty key. I will update the main code condition and test as well.

Just to confirm the change will be switching from `empty` to `is_null` like below,

**From:**
https://github.com/iamroi/laravel-api-response-builder/blob/6dc652f57e2a377e2239ea8acbd2c67ade3f325a/src/Converter.php#L149

**To:**
```
$result = is_null($cfg[ RB::KEY_KEY ]) ? $result : [$cfg[ RB::KEY_KEY ] => $result];
```

Reply:
For `null` check please use `$foo === null` instead of calling `is_null()` as this is simply faster.

Reply:
Ah just saw this comment!